### ISSUE TITLE
ie browser cannot use this lib

### ISSUE THREAD

**Issue by geniuswh** [andrei-markeev/camljs#29]

**Comment by viceice** [comment]
IE is deprecated and won't be supported in near future 🙃 

https://docs.microsoft.com/en-us/lifecycle/products/internet-explorer-11

**Comment by andrei-markeev** [comment]
Yes, but I think CamlJs should still support it for some time. There are some really slow organizations out there who still use old versions of SharePoint and even use IE11 still on corporate level. And I think it should be easy to support CamlJs in IE11, at least it worked before, so why not.

**Comment by andrei-markeev** [comment]
Actually, I created a simple test, and it seems to be working in IE11 just fine.

@geniuswh could you specify, which exact query doesn't work?
